source/updateCheck.py

```diff
@@ -245,32 +308,79 @@ def onDownloadButton(self, evt):
 		DonateRequestDialog(gui.mainFrame, self._download)
 
 	def _download(self):
-		if self.isInstalled:
-			UpdateDownloader(self.urls, fileHash=self.fileHash).start()
-		else:
-			os.startfile(self.urls[0])
+		UpdateDownloader(self.updateInfo).start()
 		self.Destroy()
 
 	def onLaterButton(self, evt):
 		state["dontRemindVersion"] = None
 		saveState()
 		self.Close()
 
+class UpdateAskInstallDialog(wx.Dialog):

```

Comment:
I have a feeling that this dialog wont have the visual spacing expected between labels and buttons. What is the easiest way to test this, so I can inspect it?

Reply:
I added some guiHelper stuff to it now. Here is a code snippet to make it show up:

```
import versionInfo
versionInfo.updateVersionType="snapshot:next"
import updateCheck
updateCheck.initialize()
updateCheck.UpdateAskInstallDialog(gui.mainFrame, "dummy_path", "2018.3")
```

Reply:
Thanks for the snippet. It looks ok to me.